### ISSUE TITLE
Support getting additional photo sizes

### ISSUE THREAD

**Issue by raiym** [slang800/instagram-screen-scrape#7]
Scraper gives only 640x640 photo urls even if photo exists in 1080x1080

**Comment by notslang** [comment]
Yeah, over in [posts.coffee:70](https://github.com/slang800/instagram-screen-scrape/blob/1f81a1ba63b4dcf78222cb46f1d9d794c8481072/lib/posts.coffee#L70), we just pick the standard resolution. However, looking at an example of the response, it seems like you can derive the other URLs from the standard resolution URL:

``` json
{
  "images": {
    "low_resolution": {
    "url": "https://scontent-ord1-1.cdninstagram.com/t51.2885-15/s320x320/e35/12531128_1720535754825405_615608517_n.jpg?ig_cache_key=MTE4MzA1OTUyNjk1ODY0ODUwMQ%3D%3D.2",
    "width": ​320,
    "height": ​320
    },
    "thumbnail": {
      "url": "https://scontent-ord1-1.cdninstagram.com/t51.2885-15/s150x150/e35/12531128_1720535754825405_615608517_n.jpg?ig_cache_key=MTE4MzA1OTUyNjk1ODY0ODUwMQ%3D%3D.2",
      "width": ​150,
      "height": ​150
    },
    "standard_resolution": {
      "url": "https://scontent-ord1-1.cdninstagram.com/t51.2885-15/s640x640/sh0.08/e35/12531128_1720535754825405_615608517_n.jpg?ig_cache_key=MTE4MzA1OTUyNjk1ODY0ODUwMQ%3D%3D.2",
      "width": ​640,
      "height": ​640
    }
  }
}
```

I would much rather give some single standard URL for the resource, or a simple media ID with a  function for formatting that ID into a URL for whatever representation of the image you want, and I'd gladly take a PR for that.

**Comment by raiym** [comment]
`JSON.stringify(rawPost.images.low_resolution.url).replace('320x320', '1080x1080')`

This worked pretty well for me. Somehow instagram stores photos in these urls even if there is no 1080px image

**Comment by raiym** [comment]
I created service backupitfor.me on top of your library. 
Any plans to add 1080x1080 support natively?

**Comment by fabiocaccamo** [comment]
I just founded and tested this scraper and it works very well. The only problem is that the image returned is not at the highest resolution.

Actually the highest resolution image url can be founded in meta property og:image, check it here: https://github.com/soimort/you-get/blob/develop/src/you_get/extractors/instagram.py
